app/bundles/LeadBundle/EventListener/DoNotContactSubscriber.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
```

Comment:
All new files are missing the file comments.

Reply:
done